src/esp/core/ManagedContainer.h

```diff
@@ -703,7 +703,7 @@ class ManagedContainer : public ManagedContainerBase {
   ManagedPtr defaultObj_ = nullptr;
 
  public:
-  ESP_SMART_POINTERS(ManagedContainer<T, Access>);
+  ESP_SMART_POINTERS(ManagedContainer<T, Access>)
```

Comment:
this ";" gives us a lot of compiling warnings. Get rid of it in this PR as well.